packages/api/models/pdrs.js

```diff
@@ -63,8 +63,7 @@ class Pdr extends Manager {
     }
 
     if (!pdr.name) {
-      log.info('Could not find name on PDR object', pdr);
-      return null;
+      throw new Error(`Could not find name on PDR object ${JSON.stringify(pdr)}`);
```

Comment:
Should we update line 62 to also throw if we want to go with this pattern? The calling code will handle it either way, but if line 62 throws instead of return null, then we could remove this line I think?

https://github.com/nasa/cumulus/pull/1316/files#diff-107068bafe2634cbe603c989e3accc8fR177

Reply:
I don't think so? Because we _try_ to generate a PDR from every message, right? So if there is no PDR object to begin with, it should just assume "this message wasn't supposed to have a PDR", whereas if there _is_ a PDR object, but it doesn't have a name, we should assume "this was meant to have a PDR, but the data provided is invalid for building a record."

Reply:
So basically you're saying that we don't want to have a fatal error logged when there isn't actually a PDR for the message, which I would agree with